docs/source/Support/bskReleaseNotes.rst

```diff
@@ -68,6 +68,8 @@ Version |release|
   simulation module
 - Added a new commanded linear force array :ref:`LinearTranslationRigidBodyMsgPayload`.
 - Added a new single-axis translating effector :ref:`linearTranslationOneDOFStateEffector`.
+- Added smoothed bang-bang and smoothed bang-coast-bang profiler options to the :ref:`prescribedRotation1DOF`
+  simulation module
```

Comment:
Add a comment to this release notes bullet point that some of the module API has changed.  Please check latest module documentation for current usage.  

This addresses the issue that we are changing how the module works without depreciating prior behavior.

Reply:
Done.